app/institutions/discover/template.hbs

```diff
@@ -1 +1,12 @@
-{{!-- placeholder route template --}}
\ No newline at end of file
+<SearchPage
+    @route='search'
+    @query={{this.q}}
+    @defaultQueryOptions={{this.defaultQueryOptions}}
+    @queryParams={{this.queryParams}}
+    @showSidePanelToggle={{this.showSidePanelToggle}}
+    @onSearch={{action this.onSearch}}
+    @resourceType={{this.resourceType}}
+    @institution={{this.model}}
+    @sort={{this.sort}}
+    @showResourceTypeFilter={{true}}
```

Comment:
This could also be a property value on the controller. Let me know should there be a preference especially if this is a property we need toggled on/off.